hawkular-datamining-api/src/test/java/org/hawkular/datamining/api/SerializationTest.java

```diff
@@ -46,7 +47,9 @@ public SerializationTest() {
     @Test
     public void testSubscription() throws IOException {
 
-        DataMiningForecaster forecaster = new DataMiningForecaster(ImmutableMetricContext.getDefault());
+        DataMiningForecaster forecaster = new DataMiningForecaster(new Metric("tenant", "feed", "foo", 1L, 2L, new
```

Comment:
I am just curious. Do you support also the feedless metrics? If there is a `null` on the `feedId` position in the constructor, does it mean the feedless metric?